CONTRIBUTING.md

```diff
@@ -36,6 +36,13 @@ You can also have `yarn` watch for changes and automatically recompile them:
 yarn watch
 ```
 
+#### Running E2E tests
+
+1. Install [K6](https://k6.io/docs/get-started/installation/)
```

Comment:
It's also can be add as a part of docker-compose

Reply:
k6 in docker introduces a lot of complexity, for now I'm content with having k6 being a requirement. In the future we can lever the dagger ci work to remove this step.